persistence/mongodb-record/src/main/scala/net/liftweb/mongodb/record/field/MongoCaseClassField.scala

```diff
@@ -83,22 +81,45 @@ class MongoCaseClassField[OwnerType <: Record[OwnerType],CaseType](rec: OwnerTyp
   }
 }
 
-class MongoCaseClassListField[OwnerType <: Record[OwnerType],CaseType](rec: OwnerType)( implicit mf: Manifest[CaseType]) extends Field[List[CaseType], OwnerType] with MandatoryTypedField[List[CaseType]] with MongoFieldFlavor[List[CaseType]] {
+class CaseClassField[OwnerType <: Record[OwnerType], CaseType](rec: OwnerType)(implicit mf: Manifest[CaseType])
+  extends CaseClassTypedField[OwnerType, CaseType](rec) with MandatoryTypedField[CaseType] {
+
+
+  def this(owner: OwnerType, value: CaseType)(implicit mf: Manifest[CaseType]) = {
+    this(owner)
+    setBox(Full(value))
+  }
+
+  override def defaultValue = null.asInstanceOf[MyType]
+}
+
+@deprecated("Use the more consistently named 'CaseClassField' instead", "3.2")
```

Comment:
Please add a note that this version will be removed in Lift 4.0.

We don't actually know if we'll have a Lift 4.0 for sure anytime soon, but good to set expectations for folks. =)